e2e/articles.spec.ts

```diff
@@ -82,7 +82,7 @@ test.describe("Authenticated Articles Page", () => {
     ).toBeVisible({ visible: !isMobile });
 
     await expect(
-      page.getByRole("link", { name: '"Codú Writing Challenge" text' }),
+      page.getByRole("link", { name: '"Find out more" text' }),
```

Comment:
```suggestion
      page.getByRole("link", { name: "Find out more" }),
```

Reply:
Thanks! Been a while since I touched Playwright and it shows 😂 Going to play with it to make sure I get gud!